ProfileManager.js

```diff
@@ -95,14 +95,26 @@ export default class ProfileManager {
 
   async createUserDocument({
     edvClient, invocationSigner, profileAgentId, referenceId,
-    content = {zcaps: {}}
+    content = {}
   }) {
     edvClient.ensureIndex({attribute: 'content.profileAgentId'});
 
+    // FIXME: content.id *must* be the profileId if this is by design
```

Comment:
`content.id` is the `id` of the User. A User's ID, at the moment, would be a Profile ID or a Profile Agent ID.